src/components/TodoItem/TodoItem.module.scss

```diff
@@ -0,0 +1,20 @@
+.item {
+  padding: 10px;
+  border: 2px solid #aaaba8;
+  border-radius: 10px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  background-color: #e0f4f5;
+
+  h3 {
+    margin-top: 0;
+    font-size: 1.5rem;
+  }
+
+  p {

```

Comment:
Better to have more specific selectors

Reply:
Done - define more srecific class selectors